docs/reference/modules/node.asciidoc

```diff
@@ -156,6 +156,13 @@ discovery.zen.minimum_master_nodes: 2 <1>
 ----------------------------
 <1> Defaults to `1`.
 
+Clusters should have at least three master-eligible nodes, with
```

Comment:
Let's start with why we are doing this: In order to be able to remain available... and finish with what should be done. And maybe "to be able" are needless words?